packages/dashboard-frontend/eslint.config.mjs

```diff
@@ -56,6 +57,26 @@ export default [
 			"no-undefined": "off",
 		},
 	},
+	{
+		files: [ ".storybook/**" ],
+		...storybookConfig[ "flat/recommended" ],
+		languageOptions: {
+			globals: {
+				...globals.node,
+			},
+		},
+		settings: {
+			// Ignore certain Storybook packages to avoid import/named errors.
```

Comment:
It seems to stem from version mismatches of @storybook/manager and @storybook/theming across different packages in the monorepo:

@storybook/manager-api is used with version ^7.6.17 in `ui-library` and `keyphrase-suggestions`.
@storybook/theming is used with versions ^8.6.12 and ^8.3.6.
The dashboard-frontend package uses @storybook/theming version ^8.6.12, which might not be compatible with the current code. 
if you rollback the versions:
```
yarn add @storybook/manager-pi@^7.6.17 @storybook/theming@^7.6.17 --dev
```
Eslint will not show an error and you can remove that ignore rule.

Or you can update all the storybook packages in the monorepo to 8.6.12 (it's not enough to update only `@storybook/theming`.

Reply:
Interesting. They are not hoisted though, due to the different versions. So if ESLint would pick up the nearest `node_modules` correctly, this would not be a problem?

Either way, I prefer to ignore the false ESLint rule instead of letting that dictate our Storybook version here or elsewhere.
It is not a matter of these two packages, they go with the rest. So that is a full upgrade or downgrade of Storybook. Quite the task for a ESLint false positive.